src/plugins/event-plugins/__integ__/WebVitalsPlugin.test.ts

```diff
@@ -80,6 +91,18 @@ test('WebVitalEvent records lcp and cls events on chrome', async (t: TestControl
     );
     const clsEventDetails = JSON.parse(clsEvents[0].details);
 
+    if (t.browser.os.name !== 'Ubuntu') {
```

Comment:
question: when would the browser os not be "Ubuntu"? would this be true only if executed via the GitHub workflow or would it also be executed in our local workspaces?

Reply:
This refers to the actual OS running on the host, and has nothing to do with the browser  (despite being in the browser details object). Unfortuantely, I couldn't figure out how to deflake the testcafe integration test for INP so I excluded it here.